aoitr.js.org

dexaroffical opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

MattIPv4 reviewed (commented):
👀 While your site does appear to have content, I'm a bit confused why it is being hosted at your GitHub account's personal GitHub Pages site, rather than in a dedicated repository on your account?

dexaroffical commented:
> 👀 While your site does appear to have content, I'm a bit confused why it is being hosted at your GitHub account's personal GitHub Pages site, rather than in a dedicated repository on your account?

This project is the Turkish version of a module. I was going to open an organization account and host it from there, but I couldn't do it, so I hosted it in a personal account.

MattIPv4 commented:
I'm not sure I follow, even if you wanted to host it on your personal account instead of an organization, you could still have it in a repository with a more logical name, like `dexaroffical/aoitr` and enable GitHub Pages there. Generally speaking, `username/username.github.io` is used to host a developer's personal site, not a random project.

dexaroffical commented:
> I'm not sure I follow, even if you wanted to host it on your personal account instead of an organization, you could still have it in a repository with a more logical name, like `dexaroffical/aoitr` and enable GitHub Pages there. Generally speaking, `username/username.github.io` is used to host a developer's personal site, not a random project.

I fixed it.

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But because you are using a workflow to deploy the page, additional steps may be required to make the CNAME work.
FYI: https://github.com/js-org/js.org/pull/7853#issuecomment-1336537858 / https://github.com/js-org/js.org/pull/8532#issuecomment-1732508431

dexaroffical commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But because you are using a workflow to deploy the page, additional steps may be required to make the CNAME work. FYI: [#7853 (comment)](https://github.com/js-org/js.org/pull/7853#issuecomment-1336537858) / [#8532 (comment)](https://github.com/js-org/js.org/pull/8532#issuecomment-1732508431)

I fixed it, the site is working, thank you

indus commented:
Welcome https://aoitr.js.org
